.changeset/itchy-impalas-fold.md

```diff
@@ -0,0 +1,10 @@
+---
+'@backstage/plugin-events-backend-module-aws-sqs': patch
+'@backstage/plugin-catalog-backend-module-aws': minor
```

Comment:
Alright it's all looking good but just one small ask. When making a breaking change, if you look at it from the user's standpoint, it's helpful to make a separate changeset for this one and declaring the breakage explicitly. Something like 

```
**BREAKING**: SomeClass.someMethod now returns a promise instead of the instance directly.
```

Happy to 🚢 after this!